### DISCUSSION TITLE
能否支持通义千问翻译

### DISCUSSION THREAD

**Discussion by zhuxiongfei** [immersive-translate/immersive-translate#1345] [Ideas]
### 新功能 | New Feature

1.支持配置通用千问API；
2.通过提示词实现中文翻译

### 为什么您想要添加这些功能 | Why you want to add these features

在中文翻译上，国产大模型准确度更高些

**Comment by theowenyoung** [comment]
目前可以使用自定义接口来接入任何官方还不支持的翻译服务：

https://immersivetranslate.com/docs/services/custom/

国内的AI模型，目前最大的问题是敏感词限制，很多外媒的文章，有30%以上的内容都是敏感词，无法翻译。
